.github/workflows/markdown-links-config.json

```diff
@@ -0,0 +1,25 @@
+{
+  "ignorePatterns": [
+    {
+        "pattern": "^http://localhost"
+    },
+    {
+        "pattern": "^https://twitter.com/"
+    },
+    {
+        "pattern": "^gaurav-nelson/"
+    }
+  ],
+  "aliveStatusCodes": [403, 429, 200],
```

Comment:
This seems wrong.

Reply:
comment from Author regarding 429 and 200
https://github.com/gaurav-nelson/github-action-markdown-link-check/issues/75#issuecomment-757593559

Reply:
My [Github action](https://github.com/SahibYar/go-libp2p/actions/runs/5433440994/jobs/9881111425#step:4:176) was showing following error
```
FILE: ./p2p/security/noise/LICENSE.md
[✖] https://opensource.org/licenses/MIT → Status: 403

5 links checked.

ERROR: 1 dead links found!
[✖] https://opensource.org/licenses/MIT → Status: 403
```
added 403 because of this https://github.com/gaurav-nelson/github-action-markdown-link-check/issues/185

Reply:
Let's use `ignorePatterns` here instead.